app/models/manageiq/providers/openstack/inventory/persister/cloud_manager.rb

```diff
@@ -40,5 +40,14 @@ def initialize_inventory_collections
         orchestration_stacks_parameters
       )
     )
+
+    add_inventory_collection(
```

Comment:
@Ladas thanks for explaining the history behind that optimization, it always helps me to know why things are the way they are.